bin/critterget.py

```diff
@@ -801,13 +818,17 @@ def getCredentials(sessionKey):
 
         # return first set of credentials
         if (debug) : print "Entities is ", entities
-        if entities is not None:
+        if entities:
             for i, c in entities.items():
```

Comment:
This is old code; I can refactor it but I'm not sure it's worth the time at this point.